### DISCUSSION TITLE
How can I lower the resolution on Diablo1 on my Vita slim?

### DISCUSSION THREAD

**Discussion by dotamann** [diasurgical/DevilutionX#2162] [Q&A] [answered]
Greetings! I was looking to do that since with the native resolution everything look so small. I checked the diablo.ini on the data folder and I couldnt find width/height settings to change. I have tried to disable fullscreen,scaling etc, and I have seen no difference. Can you guys help me? Thanks in advance.

**Comment by AJenbo** [comment, marked as answer]
The Vita port is set to exclusively render at native screen resolution. @isage I'm thinking we do not have a hardware scaler available?

**Reply by isage** [reply]
Well, it should work the same way as on other sdl2 platforms, only exception being that "window" is always 960x544. I'm gonna check if enabling resolution/scale settings breaks anything on vita.

**Reply by AJenbo** [reply]
Just removing this special case should solve it then :)

https://github.com/diasurgical/devilutionX/blob/cc9783b8d9d20d7ebd859bb005daf8adb2d8d2eb/Source/diablo.cpp#L590

**Reply by dotamann** [reply]
Thanks for the solution, but I wished I knew how to code or even use this site to implement your solution. I want to thank you for keeping diablo 1 alive through your numerous ports btw :)

**Reply by isage** [reply]
> Thanks for the solution, but I wished I knew how to code or even use this site to implement your solution. I want to thank you for keeping diablo 1 alive through your numerous ports btw :)

Don't worry, that was targeted at me)

**Comment by Johoely** [comment]
Wich port? If it's DevilutionX there's a diablo.ini file where you can set a custom resolution. I think it's only created when you build the game yourself or something. 

Or you can find an app called VitaGrafix (or something like that, found it in Homebrew Browser) that lets you change resolution settings per game. Maybe that might help. I saw this solution on [ssegold.com](https://www.ssegold.com). You can check it later.

**Reply by dotamann** [reply]
Im talking about vita port. There is a diablo.ini on vita as well but in this file there is no option to set custom resolution. VitaGrafix is not compatible with diablo vita. Some particular games use this plug in only.

**Reply by isage** [reply]
That was fixed in latest master, next release will have those options on vita too
